script.js

```diff
@@ -68,7 +74,6 @@ const main = document.querySelector('main');
 const hamburger = document.querySelector('#hamburger');
 const mobileheader = document.querySelector('.mobile-header');
 const overlay = document.querySelector('#overlay');
-// const popupwindowDesktop = document.querySelector('#popup-window-desktop-container');
 // const seeprojectDesktop = document.querySelectorAll('.see-this-project');
```

Comment:
- [ ] I kindly suggest that you remove commented out code as it doesn't make your code clean